packages/terra-action-footer/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Added
+* Added change log to the documentation on terra-dev-site
 
 1.13.0 - (June 22, 2018)
 ------------------
```

Comment:
Kinda tangential to this PR, but could you update line 42 to use the correct spelling of 'highlight'?

Reply:
Could you please comment on the line where it is spelt wrong.